packages/augur-ui/package.json

```diff
@@ -101,6 +102,19 @@
   },
   "devDependencies": {
     "@augurproject/tools": "^2.0.0-alpha.1",
+    "@babel/core": "^7.6.4",
```

Comment:
why are all these deps needed?

Reply:
They're needed by orbit-web. I tried not including them here since they're already in orbit-web/package.json but I got errors.

Reply:
we can't build orbits in it's own dir and have augur-ui reference it?

Reply:
Ah. That should work. Rely on the top-level `build` also building orbit-web then just copy everything over to its own directory in the output.

Reply:
Actually, that would involve changes to root's package.json.  The same change I removed from augur-ui, where we specifically build orbit-web.